src/ast/component.rs

```diff
@@ -42,6 +42,7 @@ impl Component {
                 | Command::Bundle(_)
                 | Command::If(_)
                 | Command::ForLoop(_)
+                | Command::PortLet(_)
```

Comment:
Port let is not needed because it is the same as a bundle of size one. #189 implements the desugaring needed to support let bound ports directly using the existing bundle infrastructure.